### PR TITLE
Added stewardship and favorite sites tables

### PR DESCRIPTION
**Pull request by florisdobber**

### CODE REVIEW

File: persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS stewardship (
+    id          SERIAL      NOT NULL PRIMARY KEY,
+    user_id     INT         NOT NULL,
+    site_id     INT         NOT NULL,
+    date_time   TIMESTAMP   DEFAULT CURRENT_TIMESTAMP,
```
Comment (liammoyn) [inline persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql:5]: Kind of weird this is date_time when the request calls it date. Would also be good to have this be more descriptive of when this time is from. Is it when the record was created or when the action was performed?

**Review by SofieDunt — COMMENTED** [review]
Looks good! Should address the documentation questions before merging

File: persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql (hunk 1 above)
Comment (florisdobber) [reply persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql]: Changed to performed_on and switched to date type

File: persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS stewardship (
+    id              SERIAL      NOT NULL PRIMARY KEY,
+    user_id         INT         NOT NULL,
+    site_id         INT         NOT NULL,
+    performed_on    DATE,
```
Comment (SofieDunt) [inline persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql:5]: Should this be `NOT NULL`?

Comment (florisdobber) [reply persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql]: Fixed

**Review by SofieDunt — APPROVED** [review]
💯

**Review by liammoyn — APPROVED** [review]

**Pull request merged**